lib/dialects/mysql/query-generator.js

```diff
@@ -3,6 +3,7 @@
 const _ = require('lodash');
```

Comment:
Some methods are copied from SQLite, SQLite inherits MySQL. Please find a way to reduce this duplicated code